client/wfnews-war/src/main/angular/src/app/components/public-incident-page/incident-info-panel-mobile/incident-info-panel-mobile.component.html

```diff
@@ -265,19 +261,19 @@ <h1>Area Restrictions</h1>
                 <h1>Suspected Cause</h1>
 
                 <div class="sub-title">
-                    <div class="sub-content">
+                    <div class="sub-content-title">
                         <div *ngIf='incident.generalIncidentCauseCatId === 1'>
-                            <img class="icon" height="24" width="24" src="/assets/images/svg-icons/human.svg"
+                            <img class="icon-top" height="24" width="24" style="padding-right: 8px !important" src="/assets/images/svg-icons/human.svg"
```

Comment:
Style looks to be duplicated and should be in a class. If `icon-top` is used elsewhere add a secondary class or a new class specific for these icons

Reply:
Created new css class